plugins/pluginmanagers/lz-n.nix

```
@@ -146,6 +149,42 @@ lib.nixvim.plugins.mkNeovimPlugin {
       };
     in
     {
+      keymaps = mkOption {
+        type = types.listOf (mkMapOptionSubmodule {
+          extraOptions.plugin = mkOption {
+            type = with types; either str lzPluginType;
+            example = "telescope.nvim";
+            description = ''
+              The name of the plugin to lazy-load when the keymap is triggered.
+              This must match the name used in the `plugins` list or be a valid plugin name that can be loaded.
+            '';
+          };
+        });
+        default = [ ];
+        example = [
+          {
+            plugin = "telescope.nvim";
+            key = "<leader>ff";
+            action = nestedLiteralLua "function() require('telescope.builtin').find_files() end";
+            options = {
+              desc = "Find files";
+            };
+          }
+          {
+            plugin = "neo-tree.nvim";
+            key = "<leader>ft";
+            action = "<CMD>Neotree toggle<CR>";
+            options = {
+              desc = "NeoTree toggle";
+            };
+          }
+        ];
+        description = ''
+          Define keymaps that will use lz.n's keymap(<plugin>).set functionality.
```

Markdown inline code:

```suggestion
          Define keymaps that will use lz.n's `keymap(<plugin>).set` functionality.
```